### ISSUE TITLE
Suggestion: reorganize navigation menu in the document (show Links/Functions etc.)

### ISSUE THREAD

**Issue by oeway** [pfnet/chainer#946]
Hi, I am new to Chainer, I think the concept is great.  

Just a little bit confused when reading the online document of chainer. 

As a regular user, what I want to know most is what kind of layers (Links) and functions is available in Chainer. But currently there is no such list available in the document, instead there are classified menus such as "learnable connections" and "loss functions" which is not intuitive for me. 

It will be surely helpful if you put all the links and functions right inside the navigation, and try to remove those redundant upper layers. Since the references are heavily used during the entire usage of chainer, it's really important to make a good navigation without too deep wrapping. (Tutorials are only for beginners, and CuPy is more for developers, thus they are not so heavily used.)

A good example for me is http://keras.io/layers/core/

What do you think?

**Comment by beam2d** [comment]
Thank you for the suggestion. I agree with you; the navigation is not good, indeed. Is it enough to just show all Links/Functions directly in the side menu, or do we still need some (better) mid-level organizations like [the numpy document](http://docs.scipy.org/doc/numpy/reference/) does?
